examples/create_table_cloud.ts

```diff
@@ -0,0 +1,27 @@
+import { createClient } from '@clickhouse/client'
+
+// ClickHouse cloud - a specific CREATE TABLE syntax is allowed
+void (async () => {
+  const client = createClient({
+    host: process.env['CLICKHOUSE_HOST'],
```

Comment:
Let's add a check the host and password aren't empty. The client doesn't validate the config and falls back to the default, which won't work with Cloud.